### ISSUE TITLE
Consider removing zoom: 1

### ISSUE THREAD

**Issue by qasimalyas** [peterramsing/lost#194]
First of all @corysimmons thanks for building lost. Its awesome and i'm using it at the moment on a SaaS codebase which is going to be a good test for this grid system.

I've been paying careful attention to the output CSS and there are things which I think can be improved.
The `zoom: 1` is redundant and unnecessary. `zoom: 1` is a proprietary rule which "fixed" layout issues in IE6 and 7 only - which was typically clearing floats.

Since `calc` isn't support in IE6, 7 and 8 then this can be omitted entirely and everything will still work from IE9+ - which is what lost supports.

I understand that there is a [calc](https://github.com/closingtag/calc-polyfill) polyfill to give IE8 support in but still, `zoom: 1`, isn't required. The traditional clearfix method is sufficient for IE8.

So in summary, please consider removing it entirely. `zoom: 1` was applicable in IE6 and 7 only.

: )

PS: I did see this issue [raised before](https://github.com/corysimmons/lost/issues/135).

**Comment by peterramsing** [comment]
I'm all for removing code if it's doing something that isn't necessary. Would you like to look into this @qasimalyas? I'm just getting a handle on what needs to be done as I've been passed this repo. If you want to put some time into code I'll help (just submit a [WIP] Pull Request and we can tackle it). I would like to plan out a greater roadmap and maybe this falls into some larger code work later.

Thoughts? :smile: 

Thanks!

**Comment by corysimmons** [comment]
[calc polyfill](https://github.com/closingtag/calc-polyfill) supports calc in IE8+ and I'd like to make sure Lost works in IE8+ with the appropriate polyfills, but you can get rid of the `zoom: 1` as it only offers IE7 support.

**Comment by qasimalyas** [comment]
@peterramsing i'd love to help - i just need to figure out a good schedule and make sure I have enough availability to spend time on this.

@corysimmons I can't vouch for how well the calc polyfill works, especially for responsive breakpoints but in principle - removing `zoom: 1` shouldn't present any layout issues. It should be ok to remove.

Will provide some details on availability to remove `zoom: 1`.

**Comment by peterramsing** [comment]
Sounds good. It might be rather simple, too. I think this might pair nicely with a 7.0.0 release.

**Comment by corysimmons** [comment]
Should literally just be removing these instances of `*zoom: 1`: https://github.com/peterramsing/lost/search?utf8=%E2%9C%93&q=zoom

**Comment by peterramsing** [comment]
@corysimmons I was thinking it was that simple but didn't know for sure. Not to steal your thunder @qasimalyas, but I ended up snagging this one. #222 There is a lot of other things to code if you want, though. Just take a look at the issues and milestones. :smile:
